packages/core/src/app/payment/paymentMethod/HostedPaymentMethod.spec.tsx

```diff
@@ -93,6 +94,24 @@ describe('HostedPaymentMethod', () => {
             .toHaveBeenCalled();
     });
 
+    it('calls onUnhandledError if initialize was failed', () => {
+        defaultProps.initializePayment = jest.fn(() => { throw new Error(); });
+
+        mount(<HostedPaymentMethodTest { ...defaultProps } />);
+
+        expect(defaultProps.onUnhandledError).toHaveBeenCalledWith(expect.any(Error));
+    });
+
+    it('calls onUnhandledError if deinitialize was failed', async () => {
+        defaultProps.deinitializePayment = jest.fn(() => {
+            throw new Error();
+        });
+
+        mount(<HostedPaymentMethodTest { ...defaultProps } />).unmount();
+
+        expect(defaultProps.onUnhandledError).toHaveBeenCalledWith(expect.any(Error));
+    });
+
```

Comment:
I can see we are testing error cases here. Could we put this file in the payment PR?

Reply:
Hey @bc-peng, sorry for the late answer, I think this Pr contains the Payment and customer code, the other one only contains the shipping code, I don't know if you added this comment in the wrong PR

Reply:
hey @bc-peng what do you mean about the payment PR?, we only have two PRs, one with the customer changes and  other wiht the shipping changes

Reply:
@AlanMunozN @BC-AOrtiz I was assuming we had a payment PR for this ticket. Apology for the misunderstanding. Please focus on other comments.